infra/feast-operator/test/e2e/e2e_test.go

```diff
@@ -72,12 +72,12 @@ var _ = Describe("controller", Ordered, func() {
 			ExpectWithOffset(1, err).NotTo(HaveOccurred())
 
 			By("building the feast image")
-			cmd = exec.Command("make", "feast-image-build")
+			cmd = exec.Command("make", "feast-ci-dev-docker-img")
 			_, err = utils.Run(cmd)
 			ExpectWithOffset(1, err).NotTo(HaveOccurred())
 
-			var feastImage = "example.com/feature-transformation-server:operator.v0"
-			By("loading the the feast image on Kind")
+			var feastImage = "feastdev/feature-server:dev"
```

Comment:
to be sure that k8s is looking locally and not trying to pull from a repo...
```suggestion
			var feastImage = "localhost/feastdev/feature-server:dev"
```

Reply:
As discussed i m tagging the image locally after building it with the image name `localhost/feastdev/feature-server:dev` and subsequent feast yaml definitions are pointing to the image `localhost/feastdev/feature-server:dev`